modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyTree.jsx

```diff
@@ -163,6 +251,25 @@ function VocabularyTree(props) {
           {currentNode}
         </div>
       </DialogContent>
+      { allowTermSelection &&
+        <DialogActions>
+          <Button color="primary"
+                  onClick={onDone}
+                  variant="contained"
+                  disabled={maxAnswers > 0 && selectedTerms.length > maxAnswers
+                         || removedTerms.length == 0
+                         || selectedTerms.filter(([a1,a2]) => !initialSelection.some(([s1,s2]) => a2 === s2)).length == 0}
```

Comment:
Now the button is always disabled when maxAnswers=1.

```suggestion
                         || removedTerms.length == 0
                            && selectedTerms.every(([a1,a2]) => initialSelection.find(([s1,s2]) => a2 === s2))}
```